content/module-reference/processing-modules/diffuse.md

```diff
@@ -91,38 +91,46 @@ The radii are expressed in pixels of the full-resolution image, so copy+pasting
 
 For electrical engineers, what is set here is a band-pass filter in wavelets space, using a gaussian frequential window centered on `central radius` with a fall-off (standard deviation) of `radius span`. Wavelet scales are analogous to harmonic frequencies and each wavelet scale defines the radius of the details to act on.
 
-## diffusion speed
+## speed (sharpen ↔ diffuse)
+
+- Positive values apply diffusion
+- Negative values undo diffusion (=sharpen)
+- Zero does nothing
 
 1st order speed (gradient)
-: The speed of diffusion of the low-frequency wavelet layers in the direction defined by the _1st order anisotropy_ setting. Positive values apply diffusion, negative values undo diffusion, zero does nothing.
+: The speed of diffusion of the low-frequency wavelet layers in the direction defined by the _1st order anisotropy_ setting. 
 
 2nd order speed (laplacian)
-: The speed of diffusion of the low-frequency wavelet layers in the direction defined by the _2nd order anisotropy_ setting. Positive values apply diffusion, negative values undo diffusion, zero does nothing.
+: The speed of diffusion of the low-frequency wavelet layers in the direction defined by the _2nd order anisotropy_ setting. 
 
 3rd order speed (gradient of laplacian)
-: The speed of diffusion of the high-frequency wavelet layers in the direction defined by the _3nd order anisotropy_ setting. Positive values apply diffusion, negative values undo diffusion, zero does nothing.
+: The speed of diffusion of the high-frequency wavelet layers in the direction defined by the _3rd order anisotropy_ setting. 
 
 4th order speed (laplacian of laplacian)
-: The speed of diffusion of the high-frequency wavelet layers in the direction defined by the _4th order anisotropy_ setting. Positive values apply diffusion, negative values undo diffusion, zero does nothing.
+: The speed of diffusion of the high-frequency wavelet layers in the direction defined by the _4th order anisotropy_ setting. 
+
+## direction
 
-## diffusion directionality
+- Positive values make diffusion rather avoid edges (isophotes)
+- Negative values make diffusion follow gradients more closely
+- Zero affects both equally (isotropic)
 
 1st order anisotropy
-: The direction of diffusion of the low-frequency wavelet layers relative to the orientation of the gradient of the low-frequency. Zero is isotropic, negative values make diffusion follow gradients more closely, positive values make diffusion follow isophotes more closely.
```

Comment:
why remove `low-frequency wavelet layers relative to the orientation of the gradient of the low-frequency` ?

Reply:
the change was according to the tooltip change, I thought it would be redundant.
I will add it again

Reply:
I added it back